pr_watch/models.py

```diff
@@ -119,12 +119,21 @@ def get_or_create_from_pr(self, pr, watched_fork):
         """
         Get or create an instance for the given pull request
         """
-        watched_pr, created = self.get_or_create(
-            fork_name=pr.fork_name,
-            branch_name=pr.branch_name,
-            github_pr_url=pr.github_pr_url,
-            watched_fork=watched_fork,
-        )
+        created = False
+        try:
+            watched_pr = WatchedPullRequest.objects.get(fork_name=pr.fork_name,
+                                                        branch_name=pr.branch_name,
+                                                        github_pr_url=pr.github_pr_url,
+                                                        watched_fork=watched_fork)
+
+        except WatchedPullRequest.DoesNotExist:
```

Comment:
Is there a reason to prefer a try / except block here, rather than get_or_create?

Reply:
If I recall, this was due to an django issue with "@properies" being access in get_or_create(), it might have been addressed in tha later revisions of Django 1.11, but I chose this fix. Without it, we get the following error:

django.core.exceptions.FieldError: Invalid field name(s) for model WatchedPullRequest: 'fork_name'.